dev/docker-compose.yaml

```diff
@@ -92,10 +92,10 @@ services:
       # - WS_OPERATION=create
       - SERVER_SECRET=secret
       - MONGO_URL=mongodb://mongodb:27017?compressors=snappy
-      - TRANSACTOR_URL=ws://transactor:3333;ws://localhost:3333
+      - TRANSACTOR_URL=ws://transactor:3333;ws://host.docker.internal:3333
```

Comment:
Let's use two variants, it is better since it will be more clear internal or external variant are used.

Reply:
Discussed